payment-menu/other-features.md

```diff
@@ -0,0 +1,1335 @@
+---
+title: Swedbank Pay Payment Menu – Other Features
+sidebar:
+  navigation:
+  - title: Payment Menu
+    items:
+    - url: /payment-menu/
+      title: Introduction
+    - url: /payment-menu/payment-order
+      title: Payment Order
+    - url: /payment-menu/capture
+      title: Capture
+    - url: /payment-menu/after-payment
+      title: After Payment
+    - url: /payment-menu/other-features
+      title: Other Features
+---
+
+{% include jumbotron.html body="After a successful integration, it may be worth
+investigating which **other features** are available in Swedbank Pay Payment Menu.
+They are listed on this very page." %}
+
+## Payment Orders
+
+{% include payment-order-get.md show_status_operations=true %}
+
+### Creating a payment order
+
+To create a payment order, you perform a `POST` request towards the
+`paymentorders` resource:
+
+{% include payment-order-purchase.md %}
+
+## Description
+
+{% include description.md %}
+
+#### Response
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)